src/CMakeLists.txt

```diff
@@ -730,9 +730,13 @@ if(ALICEVISION_HAVE_CUDA)
 
   set(CUDA_NVCC_FLAGS_DEBUG   "${CUDA_NVCC_FLAGS_DEBUG};-G;-g")
   # set(CUDA_NVCC_FLAGS_RELEASE "${CUDA_NVCC_FLAGS_RELEASE};-O3")
-  if(NOT CUDA_VERSION_MAJOR LESS 9)
-    set(ALICEVISION_CUDA_CC_LIST_BASIC 30 35 50 52 60 61 62)
-  elseif(CUDA_VERSION_MAJOR GREATER 7)
+  if(CUDA_VERSION_MAJOR GREATER_EQUAL 11)
+    set(ALICEVISION_CUDA_CC_LIST_BASIC 50 52 60 61 62 70 72 75 80)
```

Comment:
CC 50 is also on the deprecated list for SDK 11 (according to the 11.0 release notes and the warnings I'm getting).  We should either keep CC 35 as well, or drop both CC 35 and CC 50. I'm in favour of keeping 35.